src/lock/runner_service.go

```diff
@@ -0,0 +1,76 @@
+package lock
```

Comment:
This all should go in its own package. Something like `server` or `runner`

Reply:
I can't comment directly on file names but a file name should be separated with a `-` instead of an underscore. The only exception to this is for a test file when they use the `_test` suffix because that is how go finds all of the test files to run.